locales/base/translation.json

```diff
@@ -1410,19 +1410,29 @@
       "fiatExchangeSubtitle": "Fund your {{appName}} account from a bank, card or exchange.",
       "selectCurrencyTitle": "Add Funds",
       "exchangeAmountTitle": "Add {{currency}}",
-      "selectProviderHeader": "Select Payment Method"
+      "selectProviderHeader": "Select Payment Method",
+      "depositExchangeTitle": "Deposit Crypto"
     },
     "cashOut": {
       "fiatExchangeTitle": "Withdraw Funds",
       "fiatExchangeSubtitle": "Get funds out of {{appName}} and into a bank account, card, crypto exchange or to mobile money.",
       "selectCurrencyTitle": "Withdraw Funds",
       "exchangeAmountTitle": "Withdraw {{currency}}",
-      "selectProviderHeader": "Select Withdraw Method"
+      "selectProviderHeader": "Select Withdraw Method",
+      "withdrawExchangeTitle": "Withdraw Crypto"
     },
     "spend": {
       "fiatExchangeTitle": "Spend Funds",
       "fiatExchangeSubtitle": "Spend on gift cards and other purchases.",
       "selectCurrencyTitle": "Spend Funds"
+    },
+    "exchange": {
+      "copyAddress": "Copy Address",
+      "informationPartOne": "Only use exchanges or assets ",
+      "informationPartTwo": "compatible with the Celo network ",
+      "informationPartThree": "to prevent loss of funds.",
```

Comment:
Would this break up work for other languages?

Reply:
I was also wondering this; I don't know, but without breaking the text up like this in the translations file, I'm not sure how we'd be able to style only a subset like the designs require.

Reply:
could you do something like this https://github.com/valora-inc/wallet/blob/main/src/fiatconnect/LinkAccountScreen.tsx#L111-L123 where we have different stylings for different parts of this text: https://github.com/valora-inc/wallet/blob/main/locales/base/translation.json#L1302

Reply:
done! worked out of the box.